tslearn/matrix_profile.py

```diff
@@ -71,15 +71,24 @@ class MatrixProfile(TransformerMixin,
     Parameters
     ----------
     subsequence_length : int (default: 1)
-        Length of the subseries (also called window size) to be used for 
+        Length of the subseries (also called window size) to be used for
         subseries distance computations.
-    
+
+    implementation : str (default: "numpy")
+        Matrix profile implementation from the stumpy library to use for
```

Comment:
This text is not clear that one has 3 options: numpy (which does not require stumpy to be installed), stumpy and gpu_stumpy (which can work if you have some specific configuration: this should be detailed)

Reply:
@rtavenar In fact, the `stumpy` package has functions called `stump` and `gpu_stump` (note the missing "y"). So, it is `stumpy.stump` and `stumpy.gpu_stump` - basically, the naming was derived from the fact that the original published algorithms were called "stamp" and "stomp" and so, since we were using `NumPy` and `Numba`, we decided to call our implementation "stump" accordingly